### ISSUE TITLE
"end of an era" [remove legacy shortcuts code]

### ISSUE THREAD

**Issue by irulanCorrino** [clementine-player/Clementine#7336]
### i do not have `Space` anywhere in shortcuts ...i miss my shortcut and press `Space` accidentally

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Artix (arch-based distro) [Linux sov-x550cc 6.10.1-zen1-1-zen 1 ZEN SMP PREEMPT_DYNAMIC Sat, 27 Jul 2024 11:59:39 +0000 x86_64 GNU/Linux]

- Clementine version: Version 1.4 rc2

### Expected behaviour / actual behaviour
+ i do not have `Space` anywhere in shortcuts --all my shortcuts start with `Alt` AND do not include `Space` ...i miss my shortcut and press `Space` accidentally => <del>Clementine continues playing</del> `Clementine stops playing!!!` [often chaotically pausing and resuming playback]

+ the sentiment applies to `Media Buttons` [my keyboards use `Fn + XYZ` embarrasingly!]

### Steps to reproduce the problem (only for bugs)
`[SIC]`

## PROPOSED SOLUTION
remove THE BLANKING LEGACY SHORTCUTS CODE!!! i do no care Qt or DEs --i am a soprano and i am singing `radical optimism` while filing this issue!!! and my BLANKING capacitive keyboard skips my feminine touch all the time! [some dozen-ish typo ...wtf!]

<img align="right" width="44%" title="i do not have `Space` anywhere in shortcuts 2024-08-02_22-13-24" src="https://github.com/user-attachments/assets/029762a2-ba95-4c48-8495-e9cb1d585e3a">

<img align="right" width="44%" title="i do not have `Space` anywhere in shortcuts 2024-08-02_22-14-03" src="https://github.com/user-attachments/assets/497927c1-c04e-4d25-a021-90069e577726">

**Comment by matkoniecz** [comment]
Are you asking for ability to reconfigure space shortcut?

**Comment by irulanCorrino** [comment]
yes. and ideally i would like to participate (i want to get better in cpp/qt because i have my own project depending on them).

i meant to say i suspect there are some key actions in legacy code or maybe in qt framework and they compete with user-defined actions --it is not noticeable if the player is not on active workspace/window but when it is active it reacts to `space` even if i had changed play/pause shortcut to something else and `space` is not at my user shortcuts at all.

i am going to browse the player's code at github myself (for learning purposes i mentioned earlier) but i guess there are two competing code sections for player shortcuts (or something is brought by qt which thing sounds like dancing faeries, honestly)
